packages/razzle/scripts/start.js

```diff
@@ -59,7 +61,13 @@ function main() {
         stats: 'none',
       },
       /* eslint-disable no-unused-vars */
-      stats => {}
+      stats => {
+        // Just open the browser when the watch mode is first activated
+        if (!isInWatchMode) {
+          openBrowser(`http://localhost:${process.env.PORT}`);
```

Comment:
For example, my razzle project is on `https`. How to handle this case?

Reply:
`localhost` should probably be something like `process.env.HOST || options.host || 'localhost'` instead, but how are you using `https` in development? I didn't know razzle could do that.

Reply:
Alright, that's neat. I was under the impression that setting `process.env.PUBLIC_PATH` or chaning `webpack.config.output.publicPath` was only supported in the build script.

Will have to think about the protocol issue some more.

Reply:
Maybe add new env PROTOCOL?